packages/testcontainers/src/chains/reg_test_container/spv.ts

```diff
@@ -0,0 +1,31 @@
+import { RegTestContainer } from './'
+
+/**
+ * SPV testing functionnalities
```

Comment:
I guess you are going to add real(not only testing) `spv` RPCs also?
```suggestion
 * SPV RPCs for Bitcoin blockchain
```